doc/src/user_guide.rst

```diff
@@ -494,13 +498,13 @@ Parameterises the time-integration scheme used by the solver with
 
                *int*
 
-            - ``pseudo-aresid`` --- absolute residual tolerance
+            - ``pseudo-residtol`` --- pseudo residual tolerance
 
                *float*
 
-            - ``pseudo-rresid`` --- relative residual tolerance
+            - ``resid-norm`` --- pseudo residual norm
```

Comment:
Yes, pseudo-resid-norm makes more sense. It has been changed in my pull request.